packages/grafana-data/src/transformations/index.ts

```diff
@@ -15,6 +15,6 @@ export {
   ByNamesMatcherMode,
 } from './matchers/nameMatcher';
 export type { RenameByRegexTransformerOptions } from './transformers/renameByRegex';
-export { outerJoinDataFrames } from './transformers/joinDataFrames';
+export { joinDataFrames } from './transformers/joinDataFrames';
```

Comment:
```suggestion
export { joinDataFrames } from './transformers/joinDataFrames';

/** @deprecated -- will be removed in future versions */
export { joinDataFrames as outerJoinDataFrames } from './transformers/joinDataFrames';
```

Reply:
I think we still need to export the outerJoin flavor since this is in the public API

Reply:
This should fix the CI breaking change detector:  
<img width="716" alt="image" src="https://user-images.githubusercontent.com/705951/185445220-47cacd1e-1332-49b5-8881-095347acd10f.png">